src/pretix/control/sysreport.py

```diff
@@ -0,0 +1,282 @@
+            ("VALIGN", (0, 0), (-1, -1), "TOP"),
+            ("LEFTPADDING", (0, 0), (0, -1), 0),
+            ("RIGHTPADDING", (-1, 0), (-1, -1), 0),
+        ]
+        tdata = [
+            [Paragraph("Site URL:", style), Paragraph(settings.SITE_URL, style)],
+            [Paragraph("pretix version:", style), Paragraph(__version__, style)],
+            [Paragraph("Python version:", style), Paragraph(sys.version, style)],
+            [Paragraph("Platform:", style), Paragraph(platform.platform(), style)],
+            [
+                Paragraph("Database engine:", style),
+                Paragraph(settings.DATABASES["default"]["ENGINE"], style),
+            ],
+        ]
+        table = Table(tdata, colWidths=colwidths, repeatRows=0)
+        table.setStyle(TableStyle(tstyledata))
+        return table
+
+    def _usage_table(self):
+        style = self.get_style()
+        style.fontSize = 8
+        style_small = self.get_style()
+        style_small.fontSize = 6
+        style_small.leading = 8
+        style_small.alignment = TA_CENTER
+        style_small_head = self.get_style()
+        style_small_head.fontSize = 6
+        style_small_head.leading = 8
+        style_small_head.alignment = TA_CENTER
+        style_small_head.fontName = "OpenSansBd"
+
+        w = self.pagesize[0] - 30 * mm
+
+        successful = (
+            Q(status=Order.STATUS_PAID)
+            | Q(valid_if_pending=True, status=Order.STATUS_PENDING)
+            | Q(
+                Exists(
+                    OrderPayment.objects.filter(
+                        order_id=OuterRef("pk"),
+                        state__in=(
+                            OrderPayment.PAYMENT_STATE_CONFIRMED,
+                            OrderPayment.PAYMENT_STATE_REFUNDED,
+                        ),
+                    )
+                ),
+            )
+        )
+        orders_q = Order.objects.filter(
+            successful,
+            testmode=False,
+        )
+        orders_testmode_q = Order.objects.filter(
+            testmode=True,
+        )
+        orders_unconfirmed_q = Order.objects.filter(
+            ~successful,
+            testmode=False,
+        )
+        revenue_q = Transaction.objects.filter(
+            Exists(
+                OrderPayment.objects.filter(
+                    order_id=OuterRef("order_id"),
+                    state__in=(
+                        OrderPayment.PAYMENT_STATE_CONFIRMED,
+                        OrderPayment.PAYMENT_STATE_REFUNDED,
+                    ),
+                )
+            ),
+            order__testmode=False,
+        )
+
+        currencies = sorted(
+            list(
+                set(
+                    Transaction.objects.annotate(c=F("order__event__currency"))
+                    .values_list("c", flat=True)
+                    .distinct()
+                )
+            )
+        )
+
+        year_first = orders_q.aggregate(m=Min("datetime__year"))["m"] or now().year
```

Comment:
```suggestion
        year_first = orders_q.aggregate(m=Min("datetime__year"))["m"]
        if not year_first:
            year_first = now().year
        elif datetime.now().month - 1 <= self.start_month:
            year_first -= 1
```
irgendwas in der Art, sonst fehlt das erste jahr wenn der startmonat nach dem aktuellen monat liegt